start.sh

```diff
@@ -36,10 +36,6 @@ docker-compose up -d orion
 
 ## Init the chain with some state
 if [[ $SKIP_CHAIN_SETUP != 'true' ]]; then
-  set -a
-  . ./.env
-  set +a
```

Comment:
Should this be also removed from other network-tests-related bash scripts, like `run-tests.sh`, `run-full-tests.sh` etc?

Reply:
fixed in 483191dd0ad40b9561ea5ca6a801c0f0599b49b8